DEV_DOCS.md

```diff
@@ -21,6 +21,33 @@ In the future we will make this a requirement for PRs or commits.
 
 Where possible, **please rebase** your work onto master rather than merging changes from master into your PR.
 
+### Make sure tests are passing
+
+When you submit a pull request to the auspice repository, a variety of integration and unit tests will need to pass before it can be merged.
+
+You will likely want to run these tests locally before submitting:
+
+_Unit tests:_
+
+1. Install dependencies, `npm install`.
+2. Run the tests, `npm test`.
+
+_Linting:_
+
+1. Install dependencies, `npm install`.
+2. Run the linter, `npm run lint`.
+  * _If there's an issue, run `npm run lint:fix`._
+
+_Integration tests:_
+
+1. Install dependencies, `npm install`.
+2. Fetch the datasets:
+  * `npm run get-data`.
+  * `npm run get-narratives`
+3. Start the server, `npm run start`.
```

Comment:
@bcoe looks like you created `integration-test:ci` for the GH action to automatically start a server as part of the tests, which seems to work perfectly. 

Does this mean steps 3 and 4 could be consolidated (more verbose but hopefully clearer?) into something like:
> If you have a local auspice server running on port 4000 (default), run: `npm run integration-test`.
Otherwise, you can start one and run integration tests in one command with: `npm run integration-test:ci`, or start a server separately  with `npm start`.